packages/server/database/migrations/1638530068635_users_mpe_rooms.ts

```diff
@@ -6,8 +6,14 @@ export default class UsersMpeRooms extends BaseSchema {
     public up(): void {
         this.schema.createTable(this.tableName, (table) => {
             table.increments('id');
-            table.uuid('user_uuid').references('users.uuid');
-            table.uuid('mpe_room_uuid').references('mpe_rooms.uuid');
+            table
+                .uuid('user_uuid')
+                .references('users.uuid')
+                .onDelete('CASCADE');
+            table
+                .uuid('mpe_room_uuid')
+                .references('mpe_rooms.uuid')
+                .onDelete('CASCADE');
```

Comment:
This won't cause issues as we have not deployed our backend to production, but a migration should never be modified.

Reply:
To do that properly, we should create a new migration, that deletes the foreign key constraint and recreate it with the `ON DELETE` constraint you want. As shown here, without a ORM: https://stackoverflow.com/a/10356720.